src/geometry/MarchingCubes.ts

```diff
@@ -0,0 +1,318 @@
+import { vec3 } from 'gl-matrix';
+import { range } from 'lodash';
+
+import { ScalarField } from './ScalarField';
+
+const triTable: number[][] = [
```

Comment:
Some comments just explaining what this gets used for would be great (maybe a URL to an article explaining it?)

Reply:
Done.